cclib/parser/orcaparser.py

```diff
@@ -2733,19 +2766,28 @@ def energy_intensity(line: str) -> Tuple[float, float]:
             #                                   -14.444151727
             #
             # Core energy                  :    -13.604678408 Eh     -370.2021 eV
-            one_el_energy = float(next(inputfile).split()[4])  # noqa: F841
```

Comment:
These changes aren't strictly necessary but I find not using `line = next(inputfile)` annoying when trying to debug.